app/src/main/java/de/danoeh/antennapod/adapter/CoverLoader.java

```diff
@@ -132,11 +139,27 @@ protected void onResourceCleared(@Nullable Drawable placeholder) {
             ivCover.setImageDrawable(placeholder);
         }
 
-        static void setPlaceholderVisibility(TextView placeholder, boolean textAndImageCombined) {
+        static void setPlaceholderVisibility(TextView placeholder, boolean textAndImageCombined, Bitmap bitmap) {
+            boolean showTitle = UserPreferences.shouldShowSubscriptionTitle();
             if (placeholder != null) {
-                if (textAndImageCombined) {
+                if (textAndImageCombined || showTitle) {
                     int bgColor = placeholder.getContext().getResources().getColor(R.color.feed_text_bg);
-                    placeholder.setBackgroundColor(bgColor);
+                    if (bitmap == null || !showTitle) {
+                        placeholder.setBackgroundColor(bgColor);
+                        return;
+                    }
+                    Palette.from(bitmap).generate(p -> {
```

Comment:
I think this will lead to problems when scrolling quickly. When extracting the palette takes unpredictable time, we might already be showing another item in the same view when this callback is fired. We had the same problem with icons because we did not call `Glide.with( ).clear( )`.

I know it is probably pretty much to implement but using Glide to also load the palette would be a lot more clean and not have fewer edge-cases to consider. There is an example on the Glide wiki here: https://github.com/bumptech/glide/wiki/Custom-targets#palette-example